common/src/main/java/org/mcaccess/minecraftaccess/mixin/MouseHandlerAccessor.java

```diff
@@ -0,0 +1,17 @@
+package org.mcaccess.minecraftaccess.mixin;
+
+import net.minecraft.client.MouseHandler;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.gen.Invoker;
+
+@Mixin(MouseHandler.class)
+public interface MouseHandlerAccessor {
+    @Invoker("onMove")
+    void move(long windowPointer, double x, double y);
+
+    @Invoker("onPress")
+    void press(long windowPointer, int button, int action, int modifiers);
+
+    @Invoker("onScroll")
+    void scroll(long windowPointer, double xOffset, double yOffset);
```

Comment:
These should probably be renamed to `invokeOnMove`, `invokeOnPress`, and `invokeOnScroll` to avoid inconsistancy and any future confusion, also allows the annotation argument to be removed.